docs/articles/call-settings.md

```diff
@@ -1,3 +1,37 @@
 # Call settings
 
-(Chris Bacon to write this)
\ No newline at end of file
+## Introduction
+
+All C# [gRPC](http://www.grpc.io/) APIs allow customization of RPC calls by using [CallSettings](../obj/api/Google.Api.Gax.CallSettings.yml). If `CallSettings` are not specified sensible defaults are automatically provided.
+
+There are three distinct places where `CallSettings` may optionally be provided:
+
+* When calling an RPC method *(highest priority)*.
+* Client-wide when creating a gRPC client.
```

Comment:
Again, not a gRPC client. I'd probably just call it a "client object" and give an example of PublisherClient, with a link.